### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by JosephMarinier**
## Description:

Clean up unnecessary dictionary unpacking (`**` operator)

## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

File: azimuth/README.md [hunk 1]
```diff
@@ -68,10 +68,6 @@ For distributed computing we use `Dask`. In local mode, it is similar to `concur
 
 #### Known issues
 
-* [On Github Action] When a worker goes OOM, you will see errors such as
```
Comment (gabegma) [inline azimuth/README.md:71]: Any reason for deleting this?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # Custom
-.circleci
```
Comment (gabegma) [inline .gitignore:2]: Also curious why this is no longer needed

**Review by gabegma — APPROVED** [review]
LGTM! Were these changes mostly for code readability?

File: tests/test_load_config.py [hunk 3]
```diff
@@ -62,59 +62,43 @@ def test_config_shortcut():
...
-                    {"model": {"class_name": "model1"}},
-                    {"model": {"class_name": "model2"}},
-                ]
-            },
-        }
+        **MINIMAL_CONFIG,
+        pipelines=[
+            {"model": {"class_name": "model1"}},
+            {"model": {"class_name": "model2"}},
+        ],
```
Comment (JosephMarinier) [inline tests/test_load_config.py:69]: @gabegma
> Were these changes mostly for code readability?

Totally, yes.

**Comment by gabegma** [comment]
Can you rename the PR to something like `clean-up` for the merge message?

**Pull request merged**